!SESSION 2024-06-27 13:45:24.560 -----------------------------------------------
eclipse.buildId=1.9.20
java.version=17.0.4.1
java.vendor=Eclipse Adoptium
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=ko_KR
Framework arguments:  -perspective com.infineon.aurix.buildmanager.perspective.c
Command-line arguments:  -os win32 -ws win32 -arch x86_64 -perspective com.infineon.aurix.buildmanager.perspective.c

!ENTRY com.infineon.aurix.buildsystem 4 0 2024-06-27 13:45:35.548
!MESSAGE ConnectException
!STACK 0
java.net.ConnectException
	at java.net.http/jdk.internal.net.http.HttpClientImpl.send(HttpClientImpl.java:573)
	at java.net.http/jdk.internal.net.http.HttpClientFacade.send(HttpClientFacade.java:123)
	at com.infineon.aurix.buildsystem.IFXNetworkUtils$1.run(IFXNetworkUtils.java:44)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.net.ConnectException
	at java.net.http/jdk.internal.net.http.common.Utils.toConnectException(Utils.java:1047)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:198)
	at java.net.http/jdk.internal.net.http.AsyncSSLConnection.connectAsync(AsyncSSLConnection.java:56)
	at java.net.http/jdk.internal.net.http.Http1Exchange.sendHeadersAsync(Http1Exchange.java:239)
	at java.net.http/jdk.internal.net.http.Exchange.lambda$responseAsyncImpl0$9(Exchange.java:472)
	at java.net.http/jdk.internal.net.http.Exchange.checkFor407(Exchange.java:404)
	at java.net.http/jdk.internal.net.http.Exchange.lambda$responseAsyncImpl0$10(Exchange.java:476)
	at java.base/java.util.concurrent.CompletableFuture.uniHandle(CompletableFuture.java:934)
	at java.base/java.util.concurrent.CompletableFuture.uniHandleStage(CompletableFuture.java:950)
	at java.base/java.util.concurrent.CompletableFuture.handle(CompletableFuture.java:2340)
	at java.net.http/jdk.internal.net.http.Exchange.responseAsyncImpl0(Exchange.java:476)
	at java.net.http/jdk.internal.net.http.Exchange.responseAsyncImpl(Exchange.java:380)
	at java.net.http/jdk.internal.net.http.Exchange.responseAsync(Exchange.java:372)
	at java.net.http/jdk.internal.net.http.MultiExchange.responseAsyncImpl(MultiExchange.java:408)
	at java.net.http/jdk.internal.net.http.MultiExchange.lambda$responseAsyncImpl$7(MultiExchange.java:449)
	at java.base/java.util.concurrent.CompletableFuture.uniHandle(CompletableFuture.java:934)
	at java.base/java.util.concurrent.CompletableFuture.uniHandleStage(CompletableFuture.java:950)
	at java.base/java.util.concurrent.CompletableFuture.handle(CompletableFuture.java:2340)
	at java.net.http/jdk.internal.net.http.MultiExchange.responseAsyncImpl(MultiExchange.java:439)
	at java.net.http/jdk.internal.net.http.MultiExchange.lambda$responseAsync0$2(MultiExchange.java:341)
	at java.base/java.util.concurrent.CompletableFuture$UniCompose.tryFire(CompletableFuture.java:1150)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1773)
	at java.net.http/jdk.internal.net.http.HttpClientImpl$DelegatingExecutor.execute(HttpClientImpl.java:157)
	at java.base/java.util.concurrent.CompletableFuture.completeAsync(CompletableFuture.java:2673)
	at java.net.http/jdk.internal.net.http.MultiExchange.responseAsync(MultiExchange.java:294)
	at java.net.http/jdk.internal.net.http.HttpClientImpl.sendAsync(HttpClientImpl.java:654)
	at java.net.http/jdk.internal.net.http.HttpClientImpl.send(HttpClientImpl.java:552)
	... 3 more
Caused by: java.nio.channels.UnresolvedAddressException
	at java.base/sun.nio.ch.Net.checkAddress(Net.java:149)
	at java.base/sun.nio.ch.Net.checkAddress(Net.java:157)
	at java.base/sun.nio.ch.SocketChannelImpl.checkRemote(SocketChannelImpl.java:816)
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:839)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$0(PlainHttpConnection.java:183)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:569)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:185)
	... 29 more

!ENTRY com.infineon.aurix.buildsystem.remote-repo 1 0 2024-06-27 13:46:59.698
!MESSAGE Using bundled version for artifact: project-initializer:tricore-tc2xx

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 13:47:01.206
!MESSAGE Processing template: C:\IVS\7_MicroProcessorMechanism\source\.settings\aurix_TC275\Cpu0_Main.ftlc

!ENTRY org.eclipse.cdt.core 1 0 2024-06-27 13:47:20.750
!MESSAGE Indexed 'aurix_TC275' (165 sources, 638 headers) in 18 sec: 147,675 declarations; 196,219 references; 10 unresolved inclusions; 4,982 syntax errors; 5,551 unresolved names (1.6%)

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 13:49:29.903
!MESSAGE Boosting configuration TriCore Debug (TASKING) of project aurix_TC275 boosted in 1066 milliseconds. (full boost)

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 13:49:29.973
!MESSAGE Include paths were discovered in 68 milliseconds.

!ENTRY com.tasking.cdt.launch 4 107 2024-06-27 13:49:46.538
!MESSAGE 'TriCore Debug (TASKING)\aurix_TC275.elf' is not a binary file supported within this project
!SUBENTRY 1 com.tasking.cdt.launch 4 107 2024-06-27 13:49:46.538
!MESSAGE 

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 13:49:54.181
!MESSAGE Boosting configuration TriCore Debug (TASKING) of project aurix_TC275 boosted in 921 milliseconds. (incremental boost)

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 13:49:54.215
!MESSAGE Include paths were discovered in 32 milliseconds.

!ENTRY org.eclipse.cdt.dsf 4 -1 2024-06-27 13:50:19.345
!MESSAGE Invocation exception when calling a service event handler method
!STACK 0
java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.eclipse.cdt.dsf.service.DsfSession.doDispatchEvent(DsfSession.java:541)
	at org.eclipse.cdt.dsf.service.DsfSession$3.run(DsfSession.java:408)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.util.concurrent.RejectedExecutionException: Task java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask@46910eeb[Not completed, task = java.util.concurrent.Executors$RunnableAdapter@5548a337[Wrapped task = com.tasking.debug.cdt.dsf.launch.DWNServicesShutdownSequence@12d4ceaa]] rejected from org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor@55251375[Shutting down, pool size = 1, active threads = 1, queued tasks = 0, completed tasks = 1031]
	at java.base/java.util.concurrent.ThreadPoolExecutor$AbortPolicy.rejectedExecution(ThreadPoolExecutor.java:2065)
	at java.base/java.util.concurrent.ThreadPoolExecutor.reject(ThreadPoolExecutor.java:833)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.delayedExecute(ScheduledThreadPoolExecutor.java:340)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.schedule(ScheduledThreadPoolExecutor.java:562)
	at org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor.schedule(DefaultDsfExecutor.java:450)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.execute(ScheduledThreadPoolExecutor.java:705)
	at org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor.execute(DefaultDsfExecutor.java:474)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.doShutdown(DWNLaunch.java:274)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.shutdownServices(DWNLaunch.java:266)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.eventDispatched(DWNLaunch.java:195)
	... 12 more
Root exception:
java.util.concurrent.RejectedExecutionException: Task java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask@46910eeb[Not completed, task = java.util.concurrent.Executors$RunnableAdapter@5548a337[Wrapped task = com.tasking.debug.cdt.dsf.launch.DWNServicesShutdownSequence@12d4ceaa]] rejected from org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor@55251375[Shutting down, pool size = 1, active threads = 1, queued tasks = 0, completed tasks = 1031]
	at java.base/java.util.concurrent.ThreadPoolExecutor$AbortPolicy.rejectedExecution(ThreadPoolExecutor.java:2065)
	at java.base/java.util.concurrent.ThreadPoolExecutor.reject(ThreadPoolExecutor.java:833)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.delayedExecute(ScheduledThreadPoolExecutor.java:340)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.schedule(ScheduledThreadPoolExecutor.java:562)
	at org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor.schedule(DefaultDsfExecutor.java:450)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.execute(ScheduledThreadPoolExecutor.java:705)
	at org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor.execute(DefaultDsfExecutor.java:474)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.doShutdown(DWNLaunch.java:274)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.shutdownServices(DWNLaunch.java:266)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.eventDispatched(DWNLaunch.java:195)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.eclipse.cdt.dsf.service.DsfSession.doDispatchEvent(DsfSession.java:541)
	at org.eclipse.cdt.dsf.service.DsfSession$3.run(DsfSession.java:408)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)

!ENTRY org.eclipse.cdt.core 1 0 2024-06-27 13:53:15.883
!MESSAGE Indexed '01_led' (73 sources, 442 headers) in 6.77 sec: 105,240 declarations; 53,760 references; 10 unresolved inclusions; 3,910 syntax errors; 10,768 unresolved names (6.3%)

!ENTRY org.eclipse.cdt.core 1 0 2024-06-27 13:56:02.957
!MESSAGE Indexed 'ASCLIN_Shell_UART_1_KIT_TC275_SB' (165 sources, 634 headers) in 13.1 sec: 146,085 declarations; 193,550 references; 6 unresolved inclusions; 4,606 syntax errors; 5,369 unresolved names (1.6%)

!ENTRY org.eclipse.cdt.core 1 0 2024-06-27 14:27:31.439
!MESSAGE Indexed '02_uart' (165 sources, 634 headers) in 9.35 sec: 146,055 declarations; 193,439 references; 8 unresolved inclusions; 4,606 syntax errors; 5,373 unresolved names (1.6%)

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 14:34:37.662
!MESSAGE Boosting configuration Debug of project 02_uart boosted in 1882 milliseconds. (full boost)

!ENTRY com.tasking.cdt.launch 4 107 2024-06-27 14:34:57.894
!MESSAGE 'Debug\02_uart.elf' is not a binary file supported within this project
!SUBENTRY 1 com.tasking.cdt.launch 4 107 2024-06-27 14:34:57.894
!MESSAGE 

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 14:35:00.428
!MESSAGE Boosting configuration Debug of project 02_uart boosted in 379 milliseconds. (incremental boost)

!ENTRY com.tasking.cdt.launch 4 107 2024-06-27 14:35:24.489
!MESSAGE 'Debug\02_uart.elf' is not a binary file supported within this project
!SUBENTRY 1 com.tasking.cdt.launch 4 107 2024-06-27 14:35:24.489
!MESSAGE 

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 14:36:28.082
!MESSAGE Project index is not complete, and cannot be used for boosting.

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 14:36:28.083
!MESSAGE Boosting configuration Debug of project 02_uart boosted in 1 milliseconds. (incremental boost)

!ENTRY com.tasking.cdt.launch 4 107 2024-06-27 14:36:49.126
!MESSAGE 'Debug\02_uart.elf' is not a binary file supported within this project
!SUBENTRY 1 com.tasking.cdt.launch 4 107 2024-06-27 14:36:49.126
!MESSAGE 

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 14:38:32.731
!MESSAGE Boosting configuration Debug of project 02_uart boosted in 2803 milliseconds. (incremental boost)

!ENTRY com.tasking.cdt.launch 4 107 2024-06-27 14:39:27.141
!MESSAGE 'Debug\02_uart.elf' is not a binary file supported within this project
!SUBENTRY 1 com.tasking.cdt.launch 4 107 2024-06-27 14:39:27.141
!MESSAGE 

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 14:42:03.112
!MESSAGE Project index is not complete, and cannot be used for boosting.

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 14:42:03.113
!MESSAGE Boosting configuration Debug of project 02_uart boosted in 0 milliseconds. (incremental boost)

!ENTRY com.tasking.cdt.launch 4 107 2024-06-27 14:42:42.289
!MESSAGE 'Debug\02_uart.elf' is not a binary file supported within this project
!SUBENTRY 1 com.tasking.cdt.launch 4 107 2024-06-27 14:42:42.289
!MESSAGE 

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 14:43:22.520
!MESSAGE Project index is not complete, and cannot be used for boosting.

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 14:43:22.520
!MESSAGE Boosting configuration Debug of project 02_uart boosted in 0 milliseconds. (incremental boost)

!ENTRY com.tasking.cdt.launch 4 107 2024-06-27 14:43:28.214
!MESSAGE 'Debug\02_uart.elf' is not a binary file supported within this project
!SUBENTRY 1 com.tasking.cdt.launch 4 107 2024-06-27 14:43:28.214
!MESSAGE 

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 14:45:52.300
!MESSAGE Project index is not complete, and cannot be used for boosting.

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 14:45:52.301
!MESSAGE Boosting configuration Debug of project 02_uart boosted in 0 milliseconds. (incremental boost)

!ENTRY com.tasking.cdt.launch 4 107 2024-06-27 14:45:58.163
!MESSAGE 'Debug\02_uart.elf' is not a binary file supported within this project
!SUBENTRY 1 com.tasking.cdt.launch 4 107 2024-06-27 14:45:58.163
!MESSAGE 

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 14:46:29.069
!MESSAGE Project index is not complete, and cannot be used for boosting.

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 14:46:29.070
!MESSAGE Boosting configuration Debug of project 02_uart boosted in 0 milliseconds. (incremental boost)

!ENTRY com.tasking.cdt.launch 4 107 2024-06-27 14:46:34.434
!MESSAGE 'Debug\02_uart.elf' is not a binary file supported within this project
!SUBENTRY 1 com.tasking.cdt.launch 4 107 2024-06-27 14:46:34.435
!MESSAGE 

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 14:48:32.684
!MESSAGE Project index is not complete, and cannot be used for boosting.

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 14:48:32.685
!MESSAGE Boosting configuration Debug of project 02_uart boosted in 0 milliseconds. (incremental boost)

!ENTRY org.eclipse.cdt.dsf 4 -1 2024-06-27 14:52:42.952
!MESSAGE Invocation exception when calling a service event handler method
!STACK 0
java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.eclipse.cdt.dsf.service.DsfSession.doDispatchEvent(DsfSession.java:541)
	at org.eclipse.cdt.dsf.service.DsfSession$3.run(DsfSession.java:408)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.util.concurrent.RejectedExecutionException: Task java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask@42147758[Not completed, task = java.util.concurrent.Executors$RunnableAdapter@1b37fde6[Wrapped task = com.tasking.debug.cdt.dsf.launch.DWNServicesShutdownSequence@5ab21513]] rejected from org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor@32afc09[Shutting down, pool size = 1, active threads = 1, queued tasks = 0, completed tasks = 1441]
	at java.base/java.util.concurrent.ThreadPoolExecutor$AbortPolicy.rejectedExecution(ThreadPoolExecutor.java:2065)
	at java.base/java.util.concurrent.ThreadPoolExecutor.reject(ThreadPoolExecutor.java:833)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.delayedExecute(ScheduledThreadPoolExecutor.java:340)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.schedule(ScheduledThreadPoolExecutor.java:562)
	at org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor.schedule(DefaultDsfExecutor.java:450)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.execute(ScheduledThreadPoolExecutor.java:705)
	at org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor.execute(DefaultDsfExecutor.java:474)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.doShutdown(DWNLaunch.java:274)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.shutdownServices(DWNLaunch.java:266)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.eventDispatched(DWNLaunch.java:195)
	... 12 more
Root exception:
java.util.concurrent.RejectedExecutionException: Task java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask@42147758[Not completed, task = java.util.concurrent.Executors$RunnableAdapter@1b37fde6[Wrapped task = com.tasking.debug.cdt.dsf.launch.DWNServicesShutdownSequence@5ab21513]] rejected from org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor@32afc09[Shutting down, pool size = 1, active threads = 1, queued tasks = 0, completed tasks = 1441]
	at java.base/java.util.concurrent.ThreadPoolExecutor$AbortPolicy.rejectedExecution(ThreadPoolExecutor.java:2065)
	at java.base/java.util.concurrent.ThreadPoolExecutor.reject(ThreadPoolExecutor.java:833)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.delayedExecute(ScheduledThreadPoolExecutor.java:340)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.schedule(ScheduledThreadPoolExecutor.java:562)
	at org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor.schedule(DefaultDsfExecutor.java:450)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.execute(ScheduledThreadPoolExecutor.java:705)
	at org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor.execute(DefaultDsfExecutor.java:474)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.doShutdown(DWNLaunch.java:274)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.shutdownServices(DWNLaunch.java:266)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.eventDispatched(DWNLaunch.java:195)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.eclipse.cdt.dsf.service.DsfSession.doDispatchEvent(DsfSession.java:541)
	at org.eclipse.cdt.dsf.service.DsfSession$3.run(DsfSession.java:408)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 14:58:01.564
!MESSAGE Boosting configuration Debug of project 02_uart boosted in 198 milliseconds. (incremental boost)

!ENTRY org.eclipse.cdt.dsf 4 -1 2024-06-27 15:00:03.073
!MESSAGE Invocation exception when calling a service event handler method
!STACK 0
java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.eclipse.cdt.dsf.service.DsfSession.doDispatchEvent(DsfSession.java:541)
	at org.eclipse.cdt.dsf.service.DsfSession$3.run(DsfSession.java:408)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.util.concurrent.RejectedExecutionException: Task java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask@23f47dd1[Not completed, task = java.util.concurrent.Executors$RunnableAdapter@3a9d8346[Wrapped task = com.tasking.debug.cdt.dsf.launch.DWNServicesShutdownSequence@69e84872]] rejected from org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor@223a9f00[Shutting down, pool size = 1, active threads = 1, queued tasks = 0, completed tasks = 1554]
	at java.base/java.util.concurrent.ThreadPoolExecutor$AbortPolicy.rejectedExecution(ThreadPoolExecutor.java:2065)
	at java.base/java.util.concurrent.ThreadPoolExecutor.reject(ThreadPoolExecutor.java:833)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.delayedExecute(ScheduledThreadPoolExecutor.java:340)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.schedule(ScheduledThreadPoolExecutor.java:562)
	at org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor.schedule(DefaultDsfExecutor.java:450)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.execute(ScheduledThreadPoolExecutor.java:705)
	at org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor.execute(DefaultDsfExecutor.java:474)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.doShutdown(DWNLaunch.java:274)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.shutdownServices(DWNLaunch.java:266)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.eventDispatched(DWNLaunch.java:195)
	... 12 more
Root exception:
java.util.concurrent.RejectedExecutionException: Task java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask@23f47dd1[Not completed, task = java.util.concurrent.Executors$RunnableAdapter@3a9d8346[Wrapped task = com.tasking.debug.cdt.dsf.launch.DWNServicesShutdownSequence@69e84872]] rejected from org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor@223a9f00[Shutting down, pool size = 1, active threads = 1, queued tasks = 0, completed tasks = 1554]
	at java.base/java.util.concurrent.ThreadPoolExecutor$AbortPolicy.rejectedExecution(ThreadPoolExecutor.java:2065)
	at java.base/java.util.concurrent.ThreadPoolExecutor.reject(ThreadPoolExecutor.java:833)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.delayedExecute(ScheduledThreadPoolExecutor.java:340)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.schedule(ScheduledThreadPoolExecutor.java:562)
	at org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor.schedule(DefaultDsfExecutor.java:450)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.execute(ScheduledThreadPoolExecutor.java:705)
	at org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor.execute(DefaultDsfExecutor.java:474)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.doShutdown(DWNLaunch.java:274)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.shutdownServices(DWNLaunch.java:266)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.eventDispatched(DWNLaunch.java:195)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.eclipse.cdt.dsf.service.DsfSession.doDispatchEvent(DsfSession.java:541)
	at org.eclipse.cdt.dsf.service.DsfSession$3.run(DsfSession.java:408)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)

!ENTRY com.infineon.aurix.buildsystem.remote-repo 1 0 2024-06-27 15:54:47.642
!MESSAGE Using bundled version for artifact: project-initializer:tricore-tc2xx

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 15:54:49.453
!MESSAGE Processing template: C:\IVS\7_MicroProcessorMechanism\source\.settings\03_scheduling\Cpu0_Main.ftlc

!ENTRY org.eclipse.cdt.core 1 0 2024-06-27 15:55:03.452
!MESSAGE Indexed '03_scheduling' (165 sources, 638 headers) in 12.3 sec: 147,675 declarations; 196,219 references; 10 unresolved inclusions; 4,982 syntax errors; 5,551 unresolved names (1.6%)

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 16:24:59.997
!MESSAGE Boosting configuration TriCore Debug (TASKING) of project 03_scheduling boosted in 851 milliseconds. (full boost)

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 16:25:00.029
!MESSAGE Include paths were discovered in 31 milliseconds.

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 16:26:01.481
!MESSAGE Project index is not complete, and cannot be used for boosting.

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 16:26:01.481
!MESSAGE Boosting configuration TriCore Debug (TASKING) of project 03_scheduling boosted in 0 milliseconds. (incremental boost)

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 16:26:01.508
!MESSAGE Include paths were discovered in 26 milliseconds.

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 16:27:12.222
!MESSAGE Boosting configuration TriCore Debug (TASKING) of project 03_scheduling boosted in 851 milliseconds. (incremental boost)

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 16:27:12.253
!MESSAGE Include paths were discovered in 30 milliseconds.

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 16:30:01.017
!MESSAGE Project index is not complete, and cannot be used for boosting.

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 16:30:01.017
!MESSAGE Boosting configuration TriCore Debug (TASKING) of project 03_scheduling boosted in 0 milliseconds. (incremental boost)

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 16:30:01.046
!MESSAGE Include paths were discovered in 28 milliseconds.

!ENTRY org.eclipse.cdt.dsf 4 -1 2024-06-27 16:30:21.228
!MESSAGE Invocation exception when calling a service event handler method
!STACK 0
java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.eclipse.cdt.dsf.service.DsfSession.doDispatchEvent(DsfSession.java:541)
	at org.eclipse.cdt.dsf.service.DsfSession$3.run(DsfSession.java:408)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.util.concurrent.RejectedExecutionException: Task java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask@54362871[Not completed, task = java.util.concurrent.Executors$RunnableAdapter@61ab736d[Wrapped task = com.tasking.debug.cdt.dsf.launch.DWNServicesShutdownSequence@155d4176]] rejected from org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor@762ae1b9[Shutting down, pool size = 1, active threads = 1, queued tasks = 0, completed tasks = 1421]
	at java.base/java.util.concurrent.ThreadPoolExecutor$AbortPolicy.rejectedExecution(ThreadPoolExecutor.java:2065)
	at java.base/java.util.concurrent.ThreadPoolExecutor.reject(ThreadPoolExecutor.java:833)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.delayedExecute(ScheduledThreadPoolExecutor.java:340)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.schedule(ScheduledThreadPoolExecutor.java:562)
	at org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor.schedule(DefaultDsfExecutor.java:450)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.execute(ScheduledThreadPoolExecutor.java:705)
	at org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor.execute(DefaultDsfExecutor.java:474)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.doShutdown(DWNLaunch.java:274)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.shutdownServices(DWNLaunch.java:266)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.eventDispatched(DWNLaunch.java:195)
	... 12 more
Root exception:
java.util.concurrent.RejectedExecutionException: Task java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask@54362871[Not completed, task = java.util.concurrent.Executors$RunnableAdapter@61ab736d[Wrapped task = com.tasking.debug.cdt.dsf.launch.DWNServicesShutdownSequence@155d4176]] rejected from org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor@762ae1b9[Shutting down, pool size = 1, active threads = 1, queued tasks = 0, completed tasks = 1421]
	at java.base/java.util.concurrent.ThreadPoolExecutor$AbortPolicy.rejectedExecution(ThreadPoolExecutor.java:2065)
	at java.base/java.util.concurrent.ThreadPoolExecutor.reject(ThreadPoolExecutor.java:833)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.delayedExecute(ScheduledThreadPoolExecutor.java:340)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.schedule(ScheduledThreadPoolExecutor.java:562)
	at org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor.schedule(DefaultDsfExecutor.java:450)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.execute(ScheduledThreadPoolExecutor.java:705)
	at org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor.execute(DefaultDsfExecutor.java:474)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.doShutdown(DWNLaunch.java:274)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.shutdownServices(DWNLaunch.java:266)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.eventDispatched(DWNLaunch.java:195)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.eclipse.cdt.dsf.service.DsfSession.doDispatchEvent(DsfSession.java:541)
	at org.eclipse.cdt.dsf.service.DsfSession$3.run(DsfSession.java:408)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 16:35:03.462
!MESSAGE Project index is not complete, and cannot be used for boosting.

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 16:35:03.462
!MESSAGE Boosting configuration TriCore Debug (TASKING) of project 03_scheduling boosted in 0 milliseconds. (incremental boost)

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 16:35:03.493
!MESSAGE Include paths were discovered in 29 milliseconds.

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 16:36:05.312
!MESSAGE Project index is not complete, and cannot be used for boosting.

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 16:36:05.313
!MESSAGE Boosting configuration TriCore Debug (TASKING) of project 03_scheduling boosted in 0 milliseconds. (incremental boost)

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 16:36:05.334
!MESSAGE Include paths were discovered in 20 milliseconds.

!ENTRY org.eclipse.cdt.dsf 4 -1 2024-06-27 16:36:48.894
!MESSAGE Invocation exception when calling a service event handler method
!STACK 0
java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.eclipse.cdt.dsf.service.DsfSession.doDispatchEvent(DsfSession.java:541)
	at org.eclipse.cdt.dsf.service.DsfSession$3.run(DsfSession.java:408)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.util.concurrent.RejectedExecutionException: Task java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask@4083efdf[Not completed, task = java.util.concurrent.Executors$RunnableAdapter@15bd057[Wrapped task = com.tasking.debug.cdt.dsf.launch.DWNServicesShutdownSequence@429cd308]] rejected from org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor@7d96619f[Shutting down, pool size = 1, active threads = 1, queued tasks = 0, completed tasks = 1373]
	at java.base/java.util.concurrent.ThreadPoolExecutor$AbortPolicy.rejectedExecution(ThreadPoolExecutor.java:2065)
	at java.base/java.util.concurrent.ThreadPoolExecutor.reject(ThreadPoolExecutor.java:833)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.delayedExecute(ScheduledThreadPoolExecutor.java:340)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.schedule(ScheduledThreadPoolExecutor.java:562)
	at org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor.schedule(DefaultDsfExecutor.java:450)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.execute(ScheduledThreadPoolExecutor.java:705)
	at org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor.execute(DefaultDsfExecutor.java:474)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.doShutdown(DWNLaunch.java:274)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.shutdownServices(DWNLaunch.java:266)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.eventDispatched(DWNLaunch.java:195)
	... 12 more
Root exception:
java.util.concurrent.RejectedExecutionException: Task java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask@4083efdf[Not completed, task = java.util.concurrent.Executors$RunnableAdapter@15bd057[Wrapped task = com.tasking.debug.cdt.dsf.launch.DWNServicesShutdownSequence@429cd308]] rejected from org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor@7d96619f[Shutting down, pool size = 1, active threads = 1, queued tasks = 0, completed tasks = 1373]
	at java.base/java.util.concurrent.ThreadPoolExecutor$AbortPolicy.rejectedExecution(ThreadPoolExecutor.java:2065)
	at java.base/java.util.concurrent.ThreadPoolExecutor.reject(ThreadPoolExecutor.java:833)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.delayedExecute(ScheduledThreadPoolExecutor.java:340)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.schedule(ScheduledThreadPoolExecutor.java:562)
	at org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor.schedule(DefaultDsfExecutor.java:450)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.execute(ScheduledThreadPoolExecutor.java:705)
	at org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor.execute(DefaultDsfExecutor.java:474)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.doShutdown(DWNLaunch.java:274)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.shutdownServices(DWNLaunch.java:266)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.eventDispatched(DWNLaunch.java:195)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.eclipse.cdt.dsf.service.DsfSession.doDispatchEvent(DsfSession.java:541)
	at org.eclipse.cdt.dsf.service.DsfSession$3.run(DsfSession.java:408)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 16:37:19.329
!MESSAGE Project index is not complete, and cannot be used for boosting.

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 16:37:19.329
!MESSAGE Boosting configuration TriCore Debug (TASKING) of project 03_scheduling boosted in 0 milliseconds. (incremental boost)

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 16:37:19.356
!MESSAGE Include paths were discovered in 26 milliseconds.

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 16:37:47.315
!MESSAGE Project index is not complete, and cannot be used for boosting.

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 16:37:47.316
!MESSAGE Boosting configuration TriCore Debug (TASKING) of project 03_scheduling boosted in 0 milliseconds. (incremental boost)

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 16:37:47.345
!MESSAGE Include paths were discovered in 27 milliseconds.

!ENTRY org.eclipse.cdt.dsf 4 -1 2024-06-27 16:38:51.748
!MESSAGE Invocation exception when calling a service event handler method
!STACK 0
java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.eclipse.cdt.dsf.service.DsfSession.doDispatchEvent(DsfSession.java:541)
	at org.eclipse.cdt.dsf.service.DsfSession$3.run(DsfSession.java:408)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.util.concurrent.RejectedExecutionException: Task java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask@2c2d5dce[Not completed, task = java.util.concurrent.Executors$RunnableAdapter@5ec8cf8e[Wrapped task = com.tasking.debug.cdt.dsf.launch.DWNServicesShutdownSequence@55d2a004]] rejected from org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor@5b6b085[Shutting down, pool size = 1, active threads = 1, queued tasks = 0, completed tasks = 1658]
	at java.base/java.util.concurrent.ThreadPoolExecutor$AbortPolicy.rejectedExecution(ThreadPoolExecutor.java:2065)
	at java.base/java.util.concurrent.ThreadPoolExecutor.reject(ThreadPoolExecutor.java:833)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.delayedExecute(ScheduledThreadPoolExecutor.java:340)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.schedule(ScheduledThreadPoolExecutor.java:562)
	at org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor.schedule(DefaultDsfExecutor.java:450)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.execute(ScheduledThreadPoolExecutor.java:705)
	at org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor.execute(DefaultDsfExecutor.java:474)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.doShutdown(DWNLaunch.java:274)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.shutdownServices(DWNLaunch.java:266)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.eventDispatched(DWNLaunch.java:195)
	... 12 more
Root exception:
java.util.concurrent.RejectedExecutionException: Task java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask@2c2d5dce[Not completed, task = java.util.concurrent.Executors$RunnableAdapter@5ec8cf8e[Wrapped task = com.tasking.debug.cdt.dsf.launch.DWNServicesShutdownSequence@55d2a004]] rejected from org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor@5b6b085[Shutting down, pool size = 1, active threads = 1, queued tasks = 0, completed tasks = 1658]
	at java.base/java.util.concurrent.ThreadPoolExecutor$AbortPolicy.rejectedExecution(ThreadPoolExecutor.java:2065)
	at java.base/java.util.concurrent.ThreadPoolExecutor.reject(ThreadPoolExecutor.java:833)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.delayedExecute(ScheduledThreadPoolExecutor.java:340)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.schedule(ScheduledThreadPoolExecutor.java:562)
	at org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor.schedule(DefaultDsfExecutor.java:450)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.execute(ScheduledThreadPoolExecutor.java:705)
	at org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor.execute(DefaultDsfExecutor.java:474)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.doShutdown(DWNLaunch.java:274)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.shutdownServices(DWNLaunch.java:266)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.eventDispatched(DWNLaunch.java:195)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.eclipse.cdt.dsf.service.DsfSession.doDispatchEvent(DsfSession.java:541)
	at org.eclipse.cdt.dsf.service.DsfSession$3.run(DsfSession.java:408)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 16:43:37.035
!MESSAGE Project index is not complete, and cannot be used for boosting.

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 16:43:37.035
!MESSAGE Boosting configuration TriCore Debug (TASKING) of project 03_scheduling boosted in 0 milliseconds. (incremental boost)

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 16:43:37.063
!MESSAGE Include paths were discovered in 27 milliseconds.

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 16:44:18.984
!MESSAGE Project index is not complete, and cannot be used for boosting.

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 16:44:18.984
!MESSAGE Boosting configuration TriCore Debug (TASKING) of project 03_scheduling boosted in 0 milliseconds. (incremental boost)

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 16:44:19.011
!MESSAGE Include paths were discovered in 26 milliseconds.

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 16:45:59.738
!MESSAGE Boosting configuration TriCore Debug (TASKING) of project 03_scheduling boosted in 825 milliseconds. (incremental boost)

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 16:45:59.758
!MESSAGE Include paths were discovered in 19 milliseconds.

!ENTRY org.eclipse.cdt.ui 4 4 2024-06-27 16:46:00.077
!MESSAGE Error
!STACK 1
org.eclipse.core.internal.resources.ResourceException(/03_scheduling/TriCore Debug (TASKING)/uart.d)[368]: java.lang.Exception: Resource '/03_scheduling/TriCore Debug (TASKING)/uart.d' does not exist.
	at org.eclipse.core.internal.resources.ResourceException.provideStackTrace(ResourceException.java:42)
	at org.eclipse.core.internal.resources.ResourceException.<init>(ResourceException.java:38)
	at org.eclipse.core.internal.resources.Resource.checkExists(Resource.java:330)
	at org.eclipse.core.internal.resources.Resource.checkAccessible(Resource.java:204)
	at org.eclipse.core.internal.resources.Resource.findMarkers(Resource.java:994)
	at org.eclipse.cdt.internal.ui.buildconsole.BuildConsolePage.openErrorInEditor(BuildConsolePage.java:620)
	at org.eclipse.cdt.internal.ui.buildconsole.BuildConsolePage.showError(BuildConsolePage.java:591)
	at org.eclipse.cdt.internal.ui.buildconsole.BuildConsolePage.setDocument(BuildConsolePage.java:166)
	at org.eclipse.cdt.internal.ui.buildconsole.BuildConsolePage.lambda$1(BuildConsolePage.java:186)
	at org.eclipse.swt.widgets.RunnableLock.run(RunnableLock.java:40)
	at org.eclipse.swt.widgets.Synchronizer.runAsyncMessages(Synchronizer.java:132)
	at org.eclipse.swt.widgets.Display.runAsyncMessages(Display.java:4047)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:3663)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine$5.run(PartRenderingEngine.java:1155)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:338)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine.run(PartRenderingEngine.java:1046)
	at org.eclipse.e4.ui.internal.workbench.E4Workbench.createAndRunUI(E4Workbench.java:155)
	at org.eclipse.ui.internal.Workbench.lambda$3(Workbench.java:643)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:338)
	at org.eclipse.ui.internal.Workbench.createAndRunWorkbench(Workbench.java:550)
	at org.eclipse.ui.PlatformUI.createAndRunWorkbench(PlatformUI.java:171)
	at org.eclipse.ui.internal.ide.application.IDEApplication.start(IDEApplication.java:152)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:203)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:136)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:104)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:402)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:255)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:659)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:596)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1467)
!SUBENTRY 1 org.eclipse.core.resources 4 368 2024-06-27 16:46:00.077
!MESSAGE Resource '/03_scheduling/TriCore Debug (TASKING)/uart.d' does not exist.
!STACK 0
java.lang.Exception: Resource '/03_scheduling/TriCore Debug (TASKING)/uart.d' does not exist.
	at org.eclipse.core.internal.resources.ResourceException.provideStackTrace(ResourceException.java:42)
	at org.eclipse.core.internal.resources.ResourceException.<init>(ResourceException.java:38)
	at org.eclipse.core.internal.resources.Resource.checkExists(Resource.java:330)
	at org.eclipse.core.internal.resources.Resource.checkAccessible(Resource.java:204)
	at org.eclipse.core.internal.resources.Resource.findMarkers(Resource.java:994)
	at org.eclipse.cdt.internal.ui.buildconsole.BuildConsolePage.openErrorInEditor(BuildConsolePage.java:620)
	at org.eclipse.cdt.internal.ui.buildconsole.BuildConsolePage.showError(BuildConsolePage.java:591)
	at org.eclipse.cdt.internal.ui.buildconsole.BuildConsolePage.setDocument(BuildConsolePage.java:166)
	at org.eclipse.cdt.internal.ui.buildconsole.BuildConsolePage.lambda$1(BuildConsolePage.java:186)
	at org.eclipse.swt.widgets.RunnableLock.run(RunnableLock.java:40)
	at org.eclipse.swt.widgets.Synchronizer.runAsyncMessages(Synchronizer.java:132)
	at org.eclipse.swt.widgets.Display.runAsyncMessages(Display.java:4047)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:3663)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine$5.run(PartRenderingEngine.java:1155)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:338)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine.run(PartRenderingEngine.java:1046)
	at org.eclipse.e4.ui.internal.workbench.E4Workbench.createAndRunUI(E4Workbench.java:155)
	at org.eclipse.ui.internal.Workbench.lambda$3(Workbench.java:643)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:338)
	at org.eclipse.ui.internal.Workbench.createAndRunWorkbench(Workbench.java:550)
	at org.eclipse.ui.PlatformUI.createAndRunWorkbench(PlatformUI.java:171)
	at org.eclipse.ui.internal.ide.application.IDEApplication.start(IDEApplication.java:152)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:203)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:136)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:104)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:402)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:255)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:659)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:596)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1467)

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 16:46:19.780
!MESSAGE Project index is not complete, and cannot be used for boosting.

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 16:46:19.780
!MESSAGE Boosting configuration TriCore Debug (TASKING) of project 03_scheduling boosted in 0 milliseconds. (incremental boost)

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 16:46:19.801
!MESSAGE Include paths were discovered in 19 milliseconds.

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 16:46:45.334
!MESSAGE Project index is not complete, and cannot be used for boosting.

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 16:46:45.334
!MESSAGE Boosting configuration TriCore Debug (TASKING) of project 03_scheduling boosted in 0 milliseconds. (incremental boost)

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 16:46:45.369
!MESSAGE Include paths were discovered in 34 milliseconds.

!ENTRY org.eclipse.cdt.dsf 4 -1 2024-06-27 16:48:21.655
!MESSAGE Invocation exception when calling a service event handler method
!STACK 0
java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.eclipse.cdt.dsf.service.DsfSession.doDispatchEvent(DsfSession.java:541)
	at org.eclipse.cdt.dsf.service.DsfSession$3.run(DsfSession.java:408)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.util.concurrent.RejectedExecutionException: Task java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask@9c57317[Not completed, task = java.util.concurrent.Executors$RunnableAdapter@e070300[Wrapped task = com.tasking.debug.cdt.dsf.launch.DWNServicesShutdownSequence@59afdcb0]] rejected from org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor@5891b84f[Shutting down, pool size = 1, active threads = 1, queued tasks = 0, completed tasks = 1642]
	at java.base/java.util.concurrent.ThreadPoolExecutor$AbortPolicy.rejectedExecution(ThreadPoolExecutor.java:2065)
	at java.base/java.util.concurrent.ThreadPoolExecutor.reject(ThreadPoolExecutor.java:833)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.delayedExecute(ScheduledThreadPoolExecutor.java:340)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.schedule(ScheduledThreadPoolExecutor.java:562)
	at org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor.schedule(DefaultDsfExecutor.java:450)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.execute(ScheduledThreadPoolExecutor.java:705)
	at org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor.execute(DefaultDsfExecutor.java:474)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.doShutdown(DWNLaunch.java:274)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.shutdownServices(DWNLaunch.java:266)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.eventDispatched(DWNLaunch.java:195)
	... 12 more
Root exception:
java.util.concurrent.RejectedExecutionException: Task java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask@9c57317[Not completed, task = java.util.concurrent.Executors$RunnableAdapter@e070300[Wrapped task = com.tasking.debug.cdt.dsf.launch.DWNServicesShutdownSequence@59afdcb0]] rejected from org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor@5891b84f[Shutting down, pool size = 1, active threads = 1, queued tasks = 0, completed tasks = 1642]
	at java.base/java.util.concurrent.ThreadPoolExecutor$AbortPolicy.rejectedExecution(ThreadPoolExecutor.java:2065)
	at java.base/java.util.concurrent.ThreadPoolExecutor.reject(ThreadPoolExecutor.java:833)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.delayedExecute(ScheduledThreadPoolExecutor.java:340)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.schedule(ScheduledThreadPoolExecutor.java:562)
	at org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor.schedule(DefaultDsfExecutor.java:450)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.execute(ScheduledThreadPoolExecutor.java:705)
	at org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor.execute(DefaultDsfExecutor.java:474)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.doShutdown(DWNLaunch.java:274)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.shutdownServices(DWNLaunch.java:266)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.eventDispatched(DWNLaunch.java:195)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.eclipse.cdt.dsf.service.DsfSession.doDispatchEvent(DsfSession.java:541)
	at org.eclipse.cdt.dsf.service.DsfSession$3.run(DsfSession.java:408)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 16:50:30.922
!MESSAGE Boosting configuration TriCore Debug (TASKING) of project 03_scheduling boosted in 136 milliseconds. (incremental boost)

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 16:50:30.975
!MESSAGE Include paths were discovered in 51 milliseconds.

!ENTRY org.eclipse.cdt.dsf 4 -1 2024-06-27 16:51:44.024
!MESSAGE Invocation exception when calling a service event handler method
!STACK 0
java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.eclipse.cdt.dsf.service.DsfSession.doDispatchEvent(DsfSession.java:541)
	at org.eclipse.cdt.dsf.service.DsfSession$3.run(DsfSession.java:408)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.util.concurrent.RejectedExecutionException: Task java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask@18d8dd9f[Not completed, task = java.util.concurrent.Executors$RunnableAdapter@15a91889[Wrapped task = com.tasking.debug.cdt.dsf.launch.DWNServicesShutdownSequence@ec50459]] rejected from org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor@a73add[Shutting down, pool size = 1, active threads = 1, queued tasks = 0, completed tasks = 8323]
	at java.base/java.util.concurrent.ThreadPoolExecutor$AbortPolicy.rejectedExecution(ThreadPoolExecutor.java:2065)
	at java.base/java.util.concurrent.ThreadPoolExecutor.reject(ThreadPoolExecutor.java:833)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.delayedExecute(ScheduledThreadPoolExecutor.java:340)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.schedule(ScheduledThreadPoolExecutor.java:562)
	at org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor.schedule(DefaultDsfExecutor.java:450)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.execute(ScheduledThreadPoolExecutor.java:705)
	at org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor.execute(DefaultDsfExecutor.java:474)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.doShutdown(DWNLaunch.java:274)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.shutdownServices(DWNLaunch.java:266)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.eventDispatched(DWNLaunch.java:195)
	... 12 more
Root exception:
java.util.concurrent.RejectedExecutionException: Task java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask@18d8dd9f[Not completed, task = java.util.concurrent.Executors$RunnableAdapter@15a91889[Wrapped task = com.tasking.debug.cdt.dsf.launch.DWNServicesShutdownSequence@ec50459]] rejected from org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor@a73add[Shutting down, pool size = 1, active threads = 1, queued tasks = 0, completed tasks = 8323]
	at java.base/java.util.concurrent.ThreadPoolExecutor$AbortPolicy.rejectedExecution(ThreadPoolExecutor.java:2065)
	at java.base/java.util.concurrent.ThreadPoolExecutor.reject(ThreadPoolExecutor.java:833)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.delayedExecute(ScheduledThreadPoolExecutor.java:340)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.schedule(ScheduledThreadPoolExecutor.java:562)
	at org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor.schedule(DefaultDsfExecutor.java:450)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.execute(ScheduledThreadPoolExecutor.java:705)
	at org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor.execute(DefaultDsfExecutor.java:474)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.doShutdown(DWNLaunch.java:274)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.shutdownServices(DWNLaunch.java:266)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.eventDispatched(DWNLaunch.java:195)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.eclipse.cdt.dsf.service.DsfSession.doDispatchEvent(DsfSession.java:541)
	at org.eclipse.cdt.dsf.service.DsfSession$3.run(DsfSession.java:408)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 16:52:57.995
!MESSAGE Boosting configuration TriCore Debug (TASKING) of project 03_scheduling boosted in 765 milliseconds. (incremental boost)

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 16:52:58.016
!MESSAGE Include paths were discovered in 21 milliseconds.

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 16:56:50.713
!MESSAGE Project index is not complete, and cannot be used for boosting.

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 16:56:50.714
!MESSAGE Boosting configuration TriCore Debug (TASKING) of project 03_scheduling boosted in 0 milliseconds. (incremental boost)

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 16:56:50.741
!MESSAGE Include paths were discovered in 27 milliseconds.

!ENTRY org.eclipse.cdt.dsf 4 -1 2024-06-27 16:57:30.565
!MESSAGE Invocation exception when calling a service event handler method
!STACK 0
java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.eclipse.cdt.dsf.service.DsfSession.doDispatchEvent(DsfSession.java:541)
	at org.eclipse.cdt.dsf.service.DsfSession$3.run(DsfSession.java:408)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.util.concurrent.RejectedExecutionException: Task java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask@5817cd47[Not completed, task = java.util.concurrent.Executors$RunnableAdapter@ae74b85[Wrapped task = com.tasking.debug.cdt.dsf.launch.DWNServicesShutdownSequence@1f3b134d]] rejected from org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor@d66cb2d[Shutting down, pool size = 1, active threads = 1, queued tasks = 0, completed tasks = 5849]
	at java.base/java.util.concurrent.ThreadPoolExecutor$AbortPolicy.rejectedExecution(ThreadPoolExecutor.java:2065)
	at java.base/java.util.concurrent.ThreadPoolExecutor.reject(ThreadPoolExecutor.java:833)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.delayedExecute(ScheduledThreadPoolExecutor.java:340)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.schedule(ScheduledThreadPoolExecutor.java:562)
	at org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor.schedule(DefaultDsfExecutor.java:450)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.execute(ScheduledThreadPoolExecutor.java:705)
	at org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor.execute(DefaultDsfExecutor.java:474)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.doShutdown(DWNLaunch.java:274)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.shutdownServices(DWNLaunch.java:266)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.eventDispatched(DWNLaunch.java:195)
	... 12 more
Root exception:
java.util.concurrent.RejectedExecutionException: Task java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask@5817cd47[Not completed, task = java.util.concurrent.Executors$RunnableAdapter@ae74b85[Wrapped task = com.tasking.debug.cdt.dsf.launch.DWNServicesShutdownSequence@1f3b134d]] rejected from org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor@d66cb2d[Shutting down, pool size = 1, active threads = 1, queued tasks = 0, completed tasks = 5849]
	at java.base/java.util.concurrent.ThreadPoolExecutor$AbortPolicy.rejectedExecution(ThreadPoolExecutor.java:2065)
	at java.base/java.util.concurrent.ThreadPoolExecutor.reject(ThreadPoolExecutor.java:833)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.delayedExecute(ScheduledThreadPoolExecutor.java:340)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.schedule(ScheduledThreadPoolExecutor.java:562)
	at org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor.schedule(DefaultDsfExecutor.java:450)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.execute(ScheduledThreadPoolExecutor.java:705)
	at org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor.execute(DefaultDsfExecutor.java:474)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.doShutdown(DWNLaunch.java:274)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.shutdownServices(DWNLaunch.java:266)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.eventDispatched(DWNLaunch.java:195)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.eclipse.cdt.dsf.service.DsfSession.doDispatchEvent(DsfSession.java:541)
	at org.eclipse.cdt.dsf.service.DsfSession$3.run(DsfSession.java:408)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 16:57:44.685
!MESSAGE Boosting configuration TriCore Debug (TASKING) of project 03_scheduling boosted in 122 milliseconds. (incremental boost)

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 16:57:44.708
!MESSAGE Include paths were discovered in 22 milliseconds.

!ENTRY org.eclipse.cdt.dsf 4 -1 2024-06-27 16:58:13.228
!MESSAGE Invocation exception when calling a service event handler method
!STACK 0
java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.eclipse.cdt.dsf.service.DsfSession.doDispatchEvent(DsfSession.java:541)
	at org.eclipse.cdt.dsf.service.DsfSession$3.run(DsfSession.java:408)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.util.concurrent.RejectedExecutionException: Task java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask@678bf157[Not completed, task = java.util.concurrent.Executors$RunnableAdapter@221cfa84[Wrapped task = com.tasking.debug.cdt.dsf.launch.DWNServicesShutdownSequence@14fe57f]] rejected from org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor@10d648d[Shutting down, pool size = 1, active threads = 1, queued tasks = 0, completed tasks = 5326]
	at java.base/java.util.concurrent.ThreadPoolExecutor$AbortPolicy.rejectedExecution(ThreadPoolExecutor.java:2065)
	at java.base/java.util.concurrent.ThreadPoolExecutor.reject(ThreadPoolExecutor.java:833)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.delayedExecute(ScheduledThreadPoolExecutor.java:340)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.schedule(ScheduledThreadPoolExecutor.java:562)
	at org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor.schedule(DefaultDsfExecutor.java:450)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.execute(ScheduledThreadPoolExecutor.java:705)
	at org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor.execute(DefaultDsfExecutor.java:474)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.doShutdown(DWNLaunch.java:274)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.shutdownServices(DWNLaunch.java:266)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.eventDispatched(DWNLaunch.java:195)
	... 12 more
Root exception:
java.util.concurrent.RejectedExecutionException: Task java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask@678bf157[Not completed, task = java.util.concurrent.Executors$RunnableAdapter@221cfa84[Wrapped task = com.tasking.debug.cdt.dsf.launch.DWNServicesShutdownSequence@14fe57f]] rejected from org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor@10d648d[Shutting down, pool size = 1, active threads = 1, queued tasks = 0, completed tasks = 5326]
	at java.base/java.util.concurrent.ThreadPoolExecutor$AbortPolicy.rejectedExecution(ThreadPoolExecutor.java:2065)
	at java.base/java.util.concurrent.ThreadPoolExecutor.reject(ThreadPoolExecutor.java:833)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.delayedExecute(ScheduledThreadPoolExecutor.java:340)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.schedule(ScheduledThreadPoolExecutor.java:562)
	at org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor.schedule(DefaultDsfExecutor.java:450)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor.execute(ScheduledThreadPoolExecutor.java:705)
	at org.eclipse.cdt.dsf.concurrent.DefaultDsfExecutor.execute(DefaultDsfExecutor.java:474)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.doShutdown(DWNLaunch.java:274)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.shutdownServices(DWNLaunch.java:266)
	at com.tasking.debug.cdt.dsf.launch.DWNLaunch.eventDispatched(DWNLaunch.java:195)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.eclipse.cdt.dsf.service.DsfSession.doDispatchEvent(DsfSession.java:541)
	at org.eclipse.cdt.dsf.service.DsfSession$3.run(DsfSession.java:408)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 16:58:56.232
!MESSAGE Project index is not complete, and cannot be used for boosting.

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 16:58:56.233
!MESSAGE Boosting configuration TriCore Debug (TASKING) of project 03_scheduling boosted in 0 milliseconds. (incremental boost)

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 16:58:56.253
!MESSAGE Include paths were discovered in 20 milliseconds.

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 16:59:07.452
!MESSAGE Boosting configuration TriCore Debug (TASKING) of project 03_scheduling boosted in 112 milliseconds. (incremental boost)

!ENTRY com.infineon.aurix.buildsystem 1 0 2024-06-27 16:59:07.479
!MESSAGE Include paths were discovered in 26 milliseconds.
